src/adaptors/harborfi/README.md

```
@@ -0,0 +1,127 @@
+- `pool`: Unique identifier (format: `${peggedTokenAddress}-${chain}`)
+- `chain`: Chain name (formatted using `utils.formatChain()`)
+- `project`: 'harborfi'
+- `symbol`: Pegged token symbol (e.g., 'haETH', 'haBTC')
+- `tvlUsd`: Total TVL in USD (summed across all markets for the token)
+- `apyBase`: APR percentage (lowest across all markets and pools)
+- `underlyingTokens`: Array containing the pegged token address
+- `poolMeta`: Description (e.g., 'Combined from 2 market(s)')
+
+## Protocol Information
+
+- **Documentation**: https://docs.harborfinance.io/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)